packages/use-contractkit/src/contract-kit-reducer.ts

```diff
@@ -0,0 +1,128 @@
+import { Connector, Dapp, Network } from './types';
```

Comment:
Looks like there's a lint error here with the import order.
You can have VSCode auto-organize imports on save.

Reply:
We can also add a edit the settings in the vscode folder to make this the default behavior for everyone, which is what I'd recommend.